lightning-tag/implementation-setup-tasks.md

```diff
@@ -1,3 +1,5 @@
 # Pre-Implementation Setup
 
 !> Coming soon.
+
+[comment]: # (Charlie is working on steps & screenshots. Needs to include optional auto-run disable config)
```

Comment:
Correction: autoRun is actually a parameter to `config` and defaults to true

Reply:
should that be added to the  documentation for the config object in detailed-reference.md or is it currently a "hidden parameter"?

Reply:
Okay, so that is news. 😄 

My $0.02 is we document what it is, say it defaults to true, and explain a use-case where someone may want to set it to false. 

What say you?

Reply:
Yes, I was confused because CC.js uses the DB setting but LT uses this param. I agree with Erik's plan.